pkgs/by-name/li/libfprint-2-tod1-broadcom-cv3plus/package.nix

```diff
@@ -0,0 +1,93 @@
+{
+  autoPatchelfHook,
+  fetchgit,
+  lib,
+  libfprint-tod,
+  openssl,
+  patchelfUnstable, # have to use patchelfUnstable to support --rename-dynamic-symbols
+  stdenv,
+}:
+
+# Based on the package definition of `pkgs/by-name/li/libfprint-2-tod1-broadcom`
+let
+  pname = "libfprint-2-tod1-broadcom-cv3plus";
+  version = "6.3.299-6.3.040.0";
+
+  src = fetchgit {
+    url = "git://git.launchpad.net/~oem-solutions-engineers/pc-enablement/+git/libfprint-2-tod1-broadcom-cv3plus/";
+    branchName = "ubuntu/latest";
+    rev = "ac7a8ab2318216e603c8c23b279bbad28a301d3";
+    hash = "sha256-4JwoUuvskj8GqTUQpKBECCL+jkSfxpaukqRTTVTmSLk=";
+    name = "${pname}-unpacked-${version}";
+  };
+
+  wrapperLibName = "wrapper-lib.so";
+  wrapperLibSource = "wrapper-lib.c";
+
+  # wraps `fopen()` for finding firmware files
```

Comment:
Can't we use libredirect for this or anything else? We don't really want to maintain C code in the repo if avoidable.